### ISSUE TITLE
Several small issues

### ISSUE THREAD

**Issue by rk-exxec** [yokoffing/Betterfox#140]
Hey,
I used your prebuilt user.js, but I noticed some small annoyances.

Youtube doesn't remember the Playersize after a restart of the browser. How do I fix that?

I actively used Firefox sync but even removing the lines from the user.js diodnt bring it back, I had to manually reenable the `identity.fxaccount`. Do I have to reenable all the prefs that I deleted from the user.js?

And the smoothfox part doesnt seem to do anything? At least for me.

I have FF 109.0.
Thanks

Here is my current file:
```js
//
/* You may copy+paste this file and use it as it is.
 *
 * If you make changes to your about:config while the program is running, the
 * changes will be overwritten by the user.js when the application restarts.
 *
 * To make lasting changes to preferences, you will have to edit the user.js.
 */

/****************************************************************************
 * Betterfox                                                                *
 * "Ad meliora"                                                             *
 * version: 108                                                             *
 * url: https://github.com/yokoffing/Betterfox                              *
****************************************************************************/



/** DISK AVOIDANCE ***/
user_pref("browser.cache.disk.enable", false);
user_pref("browser.privatebrowsing.forceMediaMemoryCache", true);
user_pref("browser.sessionstore.privacy_level", 2);
user_pref("browser.pagethumbnails.capturing_disabled", true);



/****************************************************************************
 * SECTION: PESKYFOX                                                        *
****************************************************************************/
/** MOZILLA UI ***/
user_pref("layout.css.prefers-color-scheme.content-override", 2);
user_pref("toolkit.legacyUserProfileCustomizations.stylesheets", true);
user_pref("app.update.suppressPrompts", true);
user_pref("accessibility.force_disabled", 1);
user_pref("browser.compactmode.show", true);
user_pref("browser.privatebrowsing.vpnpromourl", "");
user_pref("extensions.getAddons.showPane", false);
user_pref("extensions.htmlaboutaddons.recommendations.enabled", false);
user_pref("browser.shell.checkDefaultBrowser", false);
user_pref("browser.newtabpage.activity-stream.asrouter.userprefs.cfr.addons", false);
user_pref("browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features", false);
user_pref("browser.preferences.moreFromMozilla", false);
user_pref("browser.tabs.tabmanager.enabled", false);
user_pref("browser.aboutwelcome.enabled", false);
user_pref("findbar.highlightAll", true);
user_pref("middlemouse.contentLoadURL", false);
user_pref("browser.privatebrowsing.enable-new-indicator", false);

/** FULLSCREEN ***/
user_pref("full-screen-api.transition-duration.enter", "0 0");
user_pref("full-screen-api.transition-duration.leave", "0 0");
user_pref("full-screen-api.warning.delay", 0);
user_pref("full-screen-api.warning.timeout", 0);

/** URL BAR ***/

/** NEW TAB PAGE ***/
user_pref("browser.newtabpage.activity-stream.feeds.topsites", false);
user_pref("browser.newtabpage.activity-stream.feeds.section.topstories", false);

/*** POCKET ***/
user_pref("extensions.pocket.enabled", false);

/****************************************************************************
 * SECTION: FASTFOX                                                         *
****************************************************************************/
user_pref("nglayout.initialpaint.delay", 0);
user_pref("nglayout.initialpaint.delay_in_oopif", 0);
user_pref("content.notify.interval", 120000); // reset pref
user_pref("browser.startup.preXulSkeletonUI", false);

user_pref("browser.sessionstore.interval", 30000);

/** EXPERIMENTAL ***/
user_pref("layout.css.grid-template-masonry-value.enabled", true);
user_pref("dom.enable_web_task_scheduling", true);
user_pref("layout.css.animation-composition.enabled", true);

/** GFX ***/
user_pref("gfx.webrender.all", true);
user_pref("gfx.webrender.precache-shaders", true);
user_pref("gfx.webrender.compositor", true);
user_pref("layers.gpu-process.enabled", true);
user_pref("media.hardware-video-decoding.enabled", true);
user_pref("gfx.canvas.accelerated", true);
user_pref("gfx.canvas.accelerated.cache-items", 32768);
user_pref("gfx.canvas.accelerated.cache-size", 4096);
user_pref("gfx.content.skia-font-cache-size", 80);
user_pref("image.cache.size", 10485760);
user_pref("image.mem.decode_bytes_at_a_time", 131072);
user_pref("image.mem.shared.unmap.min_expiration_ms", 120000);
user_pref("media.memory_cache_max_size", 1048576);
user_pref("media.memory_caches_combined_limit_kb", 2560000);
user_pref("media.cache_readahead_limit", 9000);
user_pref("media.cache_resume_threshold", 6000);

/** BROWSER CACHE ***/
user_pref("browser.cache.memory.capacity", -1);
user_pref("browser.cache.memory.max_entry_size", 153600);

/** NETWORK ***/
user_pref("network.buffer.cache.size", 262144);
user_pref("network.buffer.cache.count", 128);
user_pref("network.dnsCacheEntries", 20000);	
user_pref("network.dnsCacheExpiration", 3600);	
user_pref("network.dnsCacheExpirationGracePeriod", 240);
user_pref("network.ssl_tokens_cache_capacity", 32768);

// PREF: WebGPU [HIGHLY EXPERIMENTAL!] [NIGHTLY]
// [1] https://bugzilla.mozilla.org/show_bug.cgi?id=1746245
// [2] https://github.com/gpuweb/gpuweb/wiki/Implementation-Status
// [3] https://hacks.mozilla.org/2020/04/experimental-webgpu-in-firefox/
// [4] https://developer.chrome.com/docs/web-platform/webgpu/
user_pref("dom.webgpu.enabled", true);
user_pref("gfx.webgpu.force-enabled", true);

/****************************************************************************
 * SECTION: SMOOTHFOX                                                       *
****************************************************************************/
// see https://github.com/yokoffing/Betterfox/blob/master/Smoothfox.js
// Enter your scrolling prefs below this line:

// user_pref("general.smoothScroll",                                       true); // DEFAULT
// user_pref("general.smoothScroll.msdPhysics.continuousMotionMaxDeltaMS", 12);
// user_pref("general.smoothScroll.msdPhysics.enabled",                    true);
// user_pref("general.smoothScroll.msdPhysics.motionBeginSpringConstant",  600);
// user_pref("general.smoothScroll.msdPhysics.regularSpringConstant",      650);
// user_pref("general.smoothScroll.msdPhysics.slowdownMinDeltaMS",         25);
// user_pref("general.smoothScroll.msdPhysics.slowdownMinDeltaRatio",      2.0);
// user_pref("general.smoothScroll.msdPhysics.slowdownSpringConstant",     250);
// user_pref("general.smoothScroll.currentVelocityWeighting",              1.0);
// user_pref("general.smoothScroll.stopDecelerationWeighting",             1.0);
// user_pref("mousewheel.default.delta_multiplier_y",                      280);

/****************************************************************************
 * START: MY OVERRIDES                                                      *
****************************************************************************/
// Enter your personal prefs below this line:

/****************************************************************************
 * END: BETTERFOX                                                           *
****************************************************************************/
```

**Comment by yokoffing** [comment]
> I used your prebuilt user.js

You are not using the prebuilt. You *started* with prebuilt and made modifications. 

> Youtube doesn't remember the Playersize after a restart of the browser.

This happens to me in Google Chrome, too? Using [Enhancer](https://addons.mozilla.org/en-US/firefox/addon/enhancer-for-youtube/) is great for stuff like this.

> And the smoothfox part doesnt seem to do anything? At least for me. 

You can activate the Smoothfox prefs just like you did with the WebGPU prefs: by removing the `//` in front of them and then restarting Firefox. (As an aside, enabling WebGPU may cause issues and is not recommended unless you're a web dev.)

> I actively used Firefox sync but even removing the lines from the user.js diodnt bring it back, I had to manually reenable the identity.fxaccount. Do I have to reenable all the prefs that I deleted from the user.js?

1. Please read https://kb.mozillazine.org/User.js_file
2. and https://github.com/yokoffing/Betterfox/issues/87.
3. Then add:

```
// PREF: restore Firefox View UI
user_pref("browser.tabs.firefox-view", enable);

// PREF: restore Firefox accounts
user_pref("identity.fxaccounts.enabled", true);

// PREF: Push API seems to be needed for Sync (not needed after v. 108)
user_pref("dom.push.enabled", true);
```
to your `MY OVERRIDES` section in the file.

Note: I don't use sync and don't know the exact ins and outs of how it works.

**Comment by rk-exxec** [comment]
So I formulated my question poorly. 
I did use the prebuilt user.js and all the issues are occurring with the unmodified version.
The Youtube player did remember the size until I added the file.

The smoothfox probably didn't work, because I already made some mods to the scrolling behavior, and your settings are almost identical. I forgot about that.

I somehow misread how the user.js file works, sorry about that, I will restore and try again.
